src/compiler/Uno.Compiler.Core/BuildEnvironment.cs

```diff
@@ -54,7 +54,7 @@ public class BuildEnvironment : LogObject, IEnvironment
             Extensions = extensions;
             Options = options;
             BundleDirectory = Path.Combine(project.CacheDirectory, "bundle");
-            CacheDirectory = Path.Combine(project.CacheDirectory, "builds", (options.BuildTarget + options.Configuration).GetHashCode().ToString("X8"));
+            CacheDirectory = Path.Combine(project.CacheDirectory, "builds", (options.BuildTarget + options.Configuration).GetHashCode().ToString("x8"));
```

Comment:
I know the commit message says this is backwards compatible, but how come it is?

Reply:
The cache files are regenerated automatically when not found, so renaming the directory has no other consequences than improving naming consistency.

Reply:
Right. So you'll end up with two cache-directories until the cache gets cleared?

Reply:
Yes.